### PR TITLE
Fixin da Ripley drill

### PR DESCRIPTION
**Pull request by Robustin**
Something in the simulated turf thingy busted the drill stuff so I made the drill stuff more like it used to be when it did the simulated turf thingy and I hope it work good.

### CODE REVIEW

**Comment by Robustin** [comment]
Ugh conflict, should I just cut and paste the whole damn code to get it to realize it's intentional?

File: code/game/mecha/equipment/tools/mining_tools.dm [hunk 1]
```diff
@@ -13,9 +11,9 @@
 /obj/item/mecha_parts/mecha_equipment/drill/action(atom/target)
 	if(!action_checks(target))
 		return
-	if(istype(target, /turf) && !istype(target, /turf/simulated))
-		return
-	if(isobj(target))
+	if(istype(target, /turf) && !istype(target, /turf))
```
Comment (Supermichael777) [inline code/game/mecha/equipment/tools/mining_tools.dm]: look at what your checking is true here. then look at what your checking is false. then slap yourself in the face for not proofing your code.

**Comment by Supermichael777** [comment]
holy shit you fucked this up. this looks like a botched attempet to buff the drills.

File: code/game/mecha/equipment/tools/mining_tools.dm (hunk 1 above)
Comment (KazeEspada) [inline code/game/mecha/equipment/tools/mining_tools.dm]: The check is no longer needed except to check for probably space.

**Comment by GunHog** [comment]
I am guessing you did not even bother to test this before uploading it?

File: code/game/mecha/equipment/tools/mining_tools.dm (hunk 1 above)
Comment (RemieRichards) [inline code/game/mecha/equipment/tools/mining_tools.dm]: probably needs to be ```!istype(target, /turf/open))```
@duncathan

Comment (duncathan) [inline code/game/mecha/equipment/tools/mining_tools.dm]: this needs to be `if(istype(target,/turf/open/space))` before the return

**Comment by KorPhaeron** [comment]
Your code was horribly outdated before you attempted to fix this, you fixed problems that did not exist

**Pull request closed without merging**